Controller/LayoutConfigController.php

```diff
@@ -13,7 +13,7 @@ class LayoutConfigController extends AbstractController
 {
     public function listAction($externalNetworkId)
     {
-        $this->isGranted(array('BUSINESS_LIST_LAYOUT_CONFIG', 'BUSINESS_MANAGE_LAYOUT_CONFIG'));
+        $this->isGranted('BUSINESS_MANAGE_LAYOUT_CONFIG');
```

Comment:
I think https://github.com/CanalTP/MttBundle/blob/master/Menu/Builder.php is not used